### DISCUSSION TITLE
What is the purpose of __ExternalActors__ folder?

### DISCUSSION THREAD

**Discussion by Attano** [Sixze/ALS-Refactored#70] [Q&A]
I may be missing something, but if the intent was to separate files that are mandatory for the system core's work from non-essential ones(e.g. those used in demonstration maps etc.), then what's the point of creating 'sublevels of importance'? I mean, I can safely delete both __ExternalActors__ _and_ ALSExtras. So why not just have keep them as one folder?

Also, why do the files inside __ExternalActors__ have seemingly random generated names?

**Comment by Sixze** [comment]
This is the "One File Per Actor" feature. Actors from Playground level are now stored in this folder instead of a monolithic `.umap` file. Unreal automatically manages this folder so I can't move it or rename files inside.
